services/pypi/pypi-downloads.service.js

```diff
@@ -77,31 +77,14 @@ module.exports = class PypiDownloads extends BaseJsonService {
     return [
       {
         title: 'PyPI - Downloads',
-        pattern: 'dd/:packageName',
+        pattern: ':period(dd|dw|dm)/:packageName',
```

Comment:
If you're going to just expose the guts of the route here, is there any advantage to aliasing it in the example rather than just using the route pattern directly?

Reply:
Updated!